demo/settings.yaml

```diff
@@ -26,7 +27,11 @@ search:
 
   # Region that we are searching for jobs within:
   province_or_state: "ON"  # NOTE: this is generally 2 characters long.
-  city: "Waterloo"  # NOTE: this is the full city / town name.
+  city: "Waterloo"  # NOTE: this is the full city / town name. If set to "remote"
```

Comment:
Would prefer if we could simply omit providing the city in the YAML, we can set its default to None, and simply not include it in the search url formation. Thought?

Reply:
Yes I think that is the best thing to do. If that is the case, 'state/province' should default to None as well because it is not a used field for UK support. For UK search, 'state/province' is not used.

I will make remote search be a separate pull request to keep things more organized.

Reply:
OK, for now lets remove the comment here and in the USA YAML.

Reply:
I reverted back to how the USA YAML and normal YAML was, except for adding the UK_ENGLISH option.